content/blog/tidymodels-2022-q3/index.Rmd

```diff
@@ -0,0 +1,248 @@
+
+- [Improvements to model specification checking in tidymodels](https://www.tidyverse.org/blog/2022/10/parsnip-checking-1-0-2/)
+- [brulee 0.2.0](https://www.tidyverse.org/blog/2022/09/brulee-0-2-0/)
+- [Announcing bundle](https://www.tidyverse.org/blog/2022/09/bundle-0-1-0/)
+- [censored 0.1.0](https://www.tidyverse.org/blog/2022/08/censored-0-1-0/)
+- [rsample 1.1.0](https://www.tidyverse.org/blog/2022/08/rsample-1-1-0/)
+
+Since [our last roundup post](https://www.tidyverse.org/blog/2022/07/tidymodels-2022-q2/), there have been CRAN releases of `r nrow(tm_data)` tidymodels packages. Here are links to their NEWS files:
+
+```{r}
+#| echo: FALSE
+#| results: asis
+
+cat(tm_data$news, sep = "\n")
+```
+
+We'll highlight two specific upgrades: one for agua and another in recipes. 
+
+
+## A big upgrade for agua
+
+With version 3.38.0.1 of the [h2o](https://cran.r-project.org/package=h2o) package, agua can now tune h2o models as if they were any other type of model engine 
+
+[h2o](https://h2o.ai) has an excellent server-based computational engine for fitting a variety of different machine learning and statistical models. The h2o server can run locally or on some external high performance computing server. The downside is that it is light on tools for feature engineering and interactive data analysis. 
+
+Using h2o with tidymodels enables users to leverage the benefits of packages like recipes along with fast, server-based parallel processing. 
+
+While the syntax for model fitting and tuning are the same as any other non-h2o model, there are different ways to parallelize the work: 
+
+* The h2o server has the ability to internally parallelize individual model computations. For example, when fitting trees, the search for the best split can be done using multiple threads. The number of threads that each model should be used is set with [h2o.init(nthreads)](https://docs.h2o.ai/h2o/latest-stable/h2o-r/docs/reference/h2o.init.html). The default (`-1`) is to use all CPUs on the host. 
+
+* When using grid search, [h2o.grid(parallelism)](https://docs.h2o.ai/h2o/latest-stable/h2o-r/docs/reference/h2o.grid.html) determines how many models the h2o server should process at the same time. The default (`1`) constrains the server to run the models sequentially.
+
+* R has external parallelization tools (such as the foreach and future packages) that can start new R processes to simultaneously do work. This would run many models in parallel. For h2o, this determines how many models that the agua package could send to the server at once. This does not appear to be constrained by the `parallelism` argument to `h2o.grid()`. 
```

Comment:
```suggestion
* R has external parallelization tools (such as the foreach and future packages) that can start new R processes to simultaneously do work. This would run many models in parallel. For h2o, this determines how many models the agua package could send to the server at once. This does not appear to be constrained by the `parallelism` argument to `h2o.grid()`. 
```